src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Collections/IntervalTree`1.cs

```diff
@@ -154,33 +153,36 @@ private bool Any<TIntrospector>(int start, int length, TestInterval<TIntrospecto
             return result.ToImmutableAndFree();
         }
 
-        private void FillWithIntervalsThatMatch<TIntrospector>(
+        private int FillWithIntervalsThatMatch<TIntrospector>(
```

Comment:
Document what this returns? (it's not obvious that it'd return the number as a side effect...)

Reply:
Updated